corpora/proverbs.dep

```diff
@@ -27,3 +28,979 @@
+"<,>"
+	"," cm @punct #3->7
+"<сокур>"
+	"сокур" adj @amod #4->5
+"<көздөн>"
+	"көз" n abl @obl #5->6
+"<жаш>"
+	"жаш" n nom @nsubj #6->7
+"<чыгат>"
+	"чык." v iv aor p3 sg @root #7->0
+"<.>"
+	"." sent @punct #8->7
+
+# sent_id = proverbs:10
+# author:id = Сыргабековна&Сабыровна:293
+# text = Көйнөк тердебесе, талаа гүлдөбөйт.
+# text_en = If the shirt does not sweat, the field does not blossom.
+# label = JNW_checked
+# label_dep = to_check
+"<Көйнөк>"
+	"көйнөк" n nom @csubj #1->2
+"<тердебесе>"
+	"терде," v iv neg gna_cond p3 sg @advcl #2->5
+"<,>"
+	"," cm @punct #3->5
+"<талаа>"
+	"талаа" n nom @nsubj #4->5
+"<гүлдөбөйт>"
+	"гүлдө." v iv neg aor p3 sg @root #5->0
+"<.>"
+	"." sent @punct #6->5
+
+# sent_id = proverbs:11
+# author:id = Сыргабековна&Сабыровна:456
+# text = Чала молдо дин бузат.
+# text_en = A poser Mullah spoils religion.
+# label = JNW_checked
+# label_dep = to_check
+"<Чала>"
+	"чала" adj @amod #1->2
+"<молдо>"
+	"молдо" n nom @nsubj #2->4
+"<дин>"
+	"дин" n nom @obj #3->4
+"<бузат>"
+	"буз." v tv aor p3 sg @root #4->0
+"<.>"
+	"." sent @punct #5->4
+
+# sent_id = proverbs:12
+# author:id = Сыргабековна&Сабыровна:488
+# text = Эрте сепсең бир күнү, эрте оросуң бир жума.
+# text_en = If you sow one day early, you reap one week early.
+# label = JNW_checked
+# label_dep = to_check
+"<Эрте>"
+	"эрте" adv @advmod #1->2
+"<сепсең>"
+	"сеп" v tv gna_cond p2 sg @advcl #2->7
+"<бир>"
+	"бир" num @nummod #3->4
+"<күнү>"
+	"күн," n px3sp nom @nmod:temp #4->2
+"<,>"
+	"," cm @punct #5->7
+"<эрте>"
+	"эрте" adv @advmod #6->7
+"<оросуң>"
+	"ор" v tv aor p2 sg @root #7->0
+"<бир>"
+	"бир" num @nummod #8->9
+"<жума>"
+	"жума." n nom @nummod:temp #9->7
+"<.>"
+	"." sent @punct #10->7
+
+# sent_id = proverbs:13
+# author:id = Сыргабековна&Сабыровна:290
+# text = Көз коркок, кол баатыр.
+# text_en = The eye is a coward, the hand is a hero.
+# label = JNW_checked
+# label_dep = to_check
+"<Көз>"
+	"көз" n nom @nsubj #1->2
+"<коркок>"
+	"коркок" adj @parataxis #2->6
+		"э," cop aor p3 sg @cop #3->2
+"<,>"
+	"," cm @punct #4->6
+"<кол>"
+	"кол" n nom @nsubj #5->6
+"<баатыр>"
+	"баатыр" adj @root #6->0
+		"э." cop aor p3 sg @cop #7->6
+"<.>"
+	"." sent @punct #8->6
+
+# sent_id = proverbs:14
+# author:id = Сыргабековна&Сабыровна:374
+# text = Пулуң болсо, кулуңмун.
+# text_en = If you have money, I am your slave.
+# label = JNW_checked
+# label_dep = to_check
+"<Пулуң>"
+	"пул" n px2sg nom @obj #1->2
+"<болсо>"
+	"болсо," v iv gna_cond p3 sg @advcl #2->4
+"<,>"
+	"," cm @punct #3->4
+"<кулуңмун>"
+	"кул" n px2sg nom @root #4->0
+		"э." cop aor p1 sg @cop #5->4
+"<.>"
+	"." sent @punct #6->4
+
+# sent_id = proverbs:15
+# author:id = Сыргабековна&Сабыровна:342
+# text = Оозу кыйшык болсо да, байдын уулу сүйлөсүн.
+# text_en = Even if his mouth is crooked, let the rich person's son speak.
+# label = JNW_checked
+# label_dep = to_check
+"<Оозу>"
+	"ооз" n px3sp nom @csubj #1->3
+"<кыйшык>"
+	"кыйшык" adj @amod #2->1
```

Comment:
This isn't `@amod`.  It might be `@xcomp`?  This is "crooked" in "if [his mouth] is crooked".  It should also be `#2->3`.